src/features/bookOffer/components/BookingEventChoices.tsx

```diff
@@ -7,49 +7,35 @@ import { BookDateChoice } from 'features/bookOffer/components/BookDateChoice'
 import { BookDuoChoice } from 'features/bookOffer/components/BookDuoChoice'
 import { BookHourChoice } from 'features/bookOffer/components/BookHourChoice'
 import { BookingDetails } from 'features/bookOffer/components/BookingDetails'
+import { BookPricesChoice } from 'features/bookOffer/components/BookPricesChoice'
 import { Step } from 'features/bookOffer/context/reducer'
 import { useBookingContext } from 'features/bookOffer/context/useBookingContext'
+import { getButtonWording } from 'features/bookOffer/helpers/bookingHelpers/bookingHelpers'
 import { useCreditForOffer } from 'features/offer/helpers/useHasEnoughCredit/useHasEnoughCredit'
-import { useFeatureFlag } from 'libs/firebase/firestore/featureFlags/useFeatureFlag'
-import { RemoteStoreFeatureFlags } from 'libs/firebase/firestore/types'
-import { theme } from 'theme'
-import { ProgressBar } from 'ui/components/bars/ProgressBar'
 import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
-import { getSpacing, Spacer, Typo } from 'ui/theme'
+import { getSpacing, Spacer } from 'ui/theme'
 
 interface Props {
   stocks: OfferStockResponse[]
+  enablePricesByCategories: boolean
   offerIsDuo?: boolean
 }
 
-export const BookingEventChoices: React.FC<Props> = ({ stocks, offerIsDuo }) => {
+export const BookingEventChoices: React.FC<Props> = ({
+  stocks,
+  enablePricesByCategories,
```

Comment:
Tu as dû passer plein de tests pour que cette valeur soit à `false`.
Pourquoi tu fais pas simplement
```suggestion
  enablePricesByCategories = false,
```